Tasks/ServiceFabricDeploy/task.json

```diff
@@ -119,6 +119,15 @@
             "helpMarkDown": "Indicates whether an upgrade will be skipped if the same application type and version already exists in the cluster, otherwise the upgrade fails during validation. If enabled, re-deployments are idempotent."
         },
         {
+            "name": "skipPackageValidation",
+            "type": "boolean",
+            "label": "Skip package validation",
```

Comment:
Long labels messes with the UI and its quite long as it is. The details should rather be in the helpMarkDown, which sadly is not being used enough for other "older" parameters of this task.